client/state/selectors/is-main-site-to-secondary-site-connection.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import { getSiteOption, isJetpackSiteMainNetworkSite, isJetpackSiteSecondaryNetworkSite } from 'state/sites/selectors';
+import { withoutHttp } from 'lib/url';
+
+/**
+ * Returns true if site with id equal to mainSiteId is the main site of site with id equal to secondarySiteId and false otherwise.
```

Comment:
Looks a little verbose, could we try to simplify? If we consider the argument names are clear enough to signify we're talking about site IDs, we could go with something like:

```
Returns true if site mainSiteId is the main site of site secondarySiteId and false otherwise.
```